x/wasm/types/proposal.go

```diff
@@ -208,6 +208,10 @@ func (p InstantiateContractProposal) ValidateBasic() error {
 	if p.CodeID == 0 {
 		return errorsmod.Wrap(sdkerrors.ErrInvalidRequest, "code id is required")
 	}
+	//this code can be removed based on the preference between silent trimming the label or validating the label
```

Comment:
Please do not provide options or todos in code comment. It is hard to follow up on this and it opens another chain for work.You can use the PR description or give review feedback on your own code.

A good code comment describes the motivation or adds a bigger context on why this makes sense.

Reply:
got it